linear_search/C/linear search.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+void main()
+{

```

Comment:
{ should almost always be at the end of the previous line  [whitespace/braces] [4]

*Origin: CPPLintBear, Section: `all.cpplint`.*